modules/nlu/src/backend/typings.ts

```diff
@@ -18,17 +19,17 @@ export interface BotState {
 
   // TODO: we keep this DS in memory because it contains an unserializable lock,
   // but this should be abstracted by the train session service.
-  trainSessions: _.Dictionary<NLU.TrainingSession>
+  trainSessions: _.Dictionary<sdk.NLU.TrainingSession>
 
   cancelTraining: (lang: string) => Promise<void>
-  needsTrainingWatcher: ListenHandle
-  modelsByLang: _.Dictionary<NLU.ModelId>
+  needsTrainingWatcher: sdk.ListenHandle
+  modelsByLang: _.Dictionary<nluCore.ModelId>
 
   modelService: ModelService
 }
 
 export interface NLUProgressEvent {
   type: 'nlu'
   botId: string
-  trainSession: NLU.TrainingSession
+  trainSession: sdk.NLU.TrainingSession
```

Comment:
TrainingSession can also be moved in `common/`, unless the SDK uses it

Reply:
yeah, it could, I don't see why it's in the SDK tbh

Reply:
Couldn't we remove the whole NLU namespace from the sdk then ?

Reply:
Maybe we can @franklevasseur  can we ?